mobile-widgets/qml/main.qml

```diff
@@ -472,6 +472,16 @@ if you have network connectivity and want to sync your data to cloud storage."),
 						}
 					}
 				}
+				Kirigami.Action{
+					icon {
+						name: ":/icons/account_circle.svg"
+					}
+					text: qsTr("Reset Subsurface Cloud password")
+					onTriggered: {
+						Qt.openUrlExternally("https://cloud.subsurface-divelog.org/passwordreset")
+						globalDrawer.close()
```

Comment:
If you reset the password you should also do:
qPrefCloudStorage::set_cloud_storage_password("")
qPrefCloudStorage::set_cloud_verification_status(CS_UNKNOWN)
Leaving the 2 systems in sync, this will activate the startPage and ask for new credentials.

but I think it would be nice if the user got OK/not OK, and a short text describing what to do next.
You can use qmlManager:set_startPageText

Reply:
So this external link isn't about changing the password - it's about recovering it / setting a new one if the user can't log in. It gets you to a website which sends you an email with a different link in it (to authenticate you), on which you then can set a new password.
I assume that this will be highly asynchronous to running the app. And most likely the user isn't correctly authenticated when using that functionality.
So I'm having a hard time guessing what the right action is. Forcing a log out (in case the user was, in fact, logged in) seems a little harsh...

Reply:
I totally agree, I thought it was a password reset, maybe the text should "Recover Subsurface Cloud password", in which case I agree with your changes.